### PR TITLE
Fix some a11y errors

### PR DESCRIPTION
**Pull request by muan**
My console was blowing up telling me to fix these issues so here they are.
1. Use `button` instead of `a` for clickable non-link
2. Give fiterable unique IDs
3. Remove weird gear icon that does nothing

:sparkles:

### CODE REVIEW

File: src/contentScript.js [hunk 1]
```diff
@@ -114,6 +114,7 @@ var __gcrExtAnswers;
 
     button.setAttribute('aria-label', 'Insert canned response');
     button.style.display = 'inline-block';
+    button.type = 'button'
```
Comment (notwaldorf) [inline src/contentScript.js]: `;` 😈

File: src/contentScript.js [hunk 2]
```diff
@@ -156,7 +157,8 @@ var __gcrExtAnswers;
     var filter = createNodeWithClass('div', 'select-menu-filters');
     var filterText = createNodeWithClass('div', 'select-menu-text-filter');
     var filterInput = createNodeWithClass('input', 'js-filterable-field js-navigation-enable form-control');
-    filterInput.id = 'gcr-ext-filter-field';
+    var uniqueID = toolbar.closest('form').querySelector('textarea').id
```
Comment (notwaldorf) [inline src/contentScript.js]: `;` soooooorry!

**Comment by muan** [comment]
Screenshot:

popup totally looks the same

**Comment by notwaldorf** [comment]
Yayyyyyyy! Thanks @muan! Best contributor would merge again.

**Pull request merged**